package.json

```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "import-meta-resolve": "^1.1.1",
     "prettier": "^1.19.1",
-    "ses": "^0.16.1",
+    "ses": "^0.18.8",
```

Comment:
This doesn't seem related to community-dev. hm.

Reply:
A separate commit might have been ideal but is probably not cost-effective now. This will change soon-ish anyway.

Thanks for the details here in the PR comments.